Testscripts/Windows/VM-HOT-RESIZE.ps1

```diff
@@ -20,8 +20,28 @@ function Main {
 
         $loadBalanceName = (Get-AzureRmLoadBalancer -ResourceGroupName $AllVMData.ResourceGroupName).Name
         $VirtualMachine = Get-AzureRmVM -ResourceGroupName $AllVMData.ResourceGroupName -Name $AllVMData.RoleName
-        for ( $i = 0; $i -le 30; $i++ ) {
+        foreach ($param in $currentTestData.TestParameters.param) {
+            if ($param -match "TestMode") {
+                $testMode = $param.Replace("TestMode=","").Replace('"',"")
+            }
+        }
+        if ($testMode -eq "economy") {
+            Write-LogInfo "The test mode is economy mode"
+            $toalTestTimes = 10
```

Comment:
what does the total* test times mean?
the condition below will not guarantee that you're going to cover all possible SKUs in the upgrade VM scenario.

Reply:
The var toalTestTimes is the sum of times to  resize the VM.